portfolio/src/main/webapp/script.js

```diff
@@ -44,3 +53,21 @@ function deletePost() {
 
   fetch(url, {method: 'POST'}).then(result => populatePosts());
 }
+
+/**
+  * Updates the score of a post in Datastore and on the DOM.
+  */
+function votePost(voteButton, direction) {
+  var score;
+  if (direction==1) {
+    score = voteButton.previousElementSibling;
+  } else {
+    score = voteButton.nextElementSibling;
+  }
```

Comment:
This if/else statement can be replaced with a ternary statement: 
`var score = direction == 1 ? voteButton.previousElementSibling : voteButton.nextElementSibling;`